crates/jstzd/tests/octez_node_test.rs

```diff
@@ -0,0 +1,53 @@
+#![cfg(not(target_os = "macos"))]
+// This test is disabled for macos because octez binary is not available
+
+use jstzd::task::{octez_node, Task};
+use rand::Rng;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+#[ignore = "ignore for now until octez binaries in CI are set up properly"]
+#[tokio::test(flavor = "multi_thread")]
+async fn octez_node_test() {
+    let tmp_dir = std::env::temp_dir();
+    let ts = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .unwrap()
+        .as_nanos();
+    let data_dir_path = tmp_dir.join(format!("jstzd-octez-node-test-{}", ts));
+    std::fs::create_dir(&data_dir_path).unwrap();
+    let log_path = tmp_dir.join(format!("jstzd-octez-node-test-{}-log.txt", ts));
+    let rpc_endpoint = format!(
+        "localhost:{}",
+        10000 + rand::thread_rng().gen_range(0..10000)
+    );
+
+    let mut config_builer = octez_node::OctezNodeConfigBuilder::new();
+    config_builer
+        .set_binary_path("octez-node")
+        .set_data_dir(data_dir_path.to_str().unwrap())
+        .set_network("sandbox")
+        .set_rpc_endpoint(&rpc_endpoint)
+        .set_log_file(log_path.to_str().unwrap())
+        .set_run_options(&[]);
+    let mut f = octez_node::OctezNode::spawn(config_builer.build().unwrap())
+        .await
+        .unwrap();
+
+    let health_check_endpoint = format!("http://{}/health/ready", rpc_endpoint);
+    // Should be able to hit the endpoint since the node should have been launched
+    tokio::time::sleep(tokio::time::Duration::from_secs(10)).await;
+    let res = reqwest::get(&health_check_endpoint).await;
```

Comment:
Maybe instead we just poll with 10 sec timeout until health check is returns success?

Reply:
Updated

Reply:
I don't see any changes. Did you forget to push the change?

Reply:
No I have pushed it

Reply:
Oops was looking at outdated!